samples/browser/src/main/java/org/mozilla/samples/browser/SampleApplication.kt

```diff
@@ -59,7 +61,14 @@ class SampleApplication : Application() {
                         val selected = components.sessionManager.findSessionById(sessionId)
                         selected?.let { components.tabsUseCases.selectTab(it) }
                 },
-                onUpdatePermissionRequest = components.addonUpdater::onUpdatePermissionRequest
+                onUpdatePermissionRequest = components.addonUpdater::onUpdatePermissionRequest,
+                onActionPopupToggleOverride = {
```

Comment:
In the sample browser, should we leave the tab to make things easier to test? Wdyt?

Reply:
That makes sense to me. I will remove right before merging